src/components/Monitoring/dashboard/PVBatteryChart.tsx

```diff
@@ -147,7 +148,7 @@ export default function PVBatteryChart(props: PVBatteryChartProps): JSX.Element
                 time: {
                   unit: getUnit(selectedPVBatteryPeriod),
                   displayFormats: {
-                    hour: 'MMM d h:mm',
+                    hour: 'PPp',
```

Comment:
With this format we are including the year that I think we didn't want to include it (because it takes more space)

Reply:
Also, when changing to gibberish, to the format that include hours doesn't look like japanese